constitution.tex

```diff
@@ -900,6 +900,7 @@ \subsection{Modifying the Constitution}
 
 \asubsection{Eligible Votes}
 The number of Active Members eligible to vote on the issue.
+The eligible voters for a vote are determined and fixed when that vote is called. 
```

Comment:
Hmm, so if someone becomes active during the ballot period for an amendment, they would not be eligible? Or how is called defined here? Called for or called as resolved?

Reply:
good point. changed wording.